modulemd/modulemd-modulemetadata.c

```diff
@@ -1,6 +1,7 @@
 /* modulemd-metadata.c
  *
  * Copyright (C) 2017 Stephen Gallagher
+ * Copyright (C) 2017 Igor Gnatenko <ignatenkobrain@fedoraproject.org>
  *
```

Comment:
Every change in this patch is unnecessary and makes it difficult to follow the rest of the changes. You removed a large number of commented-out lines that were placeholders for future work that needed to be done, changed variable names for no clear reason and reordered code. Not a single line of this appears to have been done for any valid reason. Or if it was, it is unclear and should have been justified in the commit message. This patch is rejected.